dvc/tree/base.py

```diff
@@ -48,7 +48,7 @@ class BaseTree:
     scheme = "base"
     REQUIRES = {}
     PATH_CLS = URLInfo
-    JOBS = 4 * cpu_count()
+    JOBS = 4
```

Comment:
Since multiplication of a default jobs count by the number of CPU cores caused exceeding of file descriptor limit on the machines with multiple CPU cores I see a reason to avoid the multiplication everywhere including the default class attribute.

Reply:
@I159 This is out of scope for this change. Let's keep the defaults the same they were before.

Reply:
Got it, reverting the default. 🙏🏼